apps/aecore/lib/aecore/miner/worker.ex

```diff
@@ -306,87 +283,24 @@ defmodule Aecore.Miner.Worker do
 
   ## Internal
 
+  defp get_pool_values() do
+    pool_values = Map.values(Pool.get_pool())
+    max_txs_for_block = Application.get_env(:aecore, :tx_data)[:max_txs_per_block] - 1
+
+    if Enum.empty?(pool_values) or length(pool_values) < max_txs_for_block do
+      pool_values
+    else
+      Enum.slice(pool_values, max_txs_for_block)
```

Comment:
Does the slice work here? I checked the docs for slice, it either takes starting index and amount to slice or a range from which to slice (something like 0..10) so passing 2 parameters and the 2nd being just an integer shouldn't work

Reply:
You are right, it should be 0..max_txs_for_block